arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtso

```diff
@@ -0,0 +1,36 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+    compatible = "embeddedts,ts-7250-v3";
+
+    fragment@0 {
+        target-path = "/";
+        __overlay__ {
+            i2c5_gpio: i2c-104 {
+                compatible = "i2c-gpio";
+
+                /* FPGA bank1 pins: SCL=13, SDA=8 */
+                scl-gpios = <&fpga_bank1 13 0>;
+                sda-gpios = <&fpga_bank1  8 0>;
+
+                /* Open-drain for both lines (legacy property name for broad kernel support) */
+                i2c-gpio,open-drain;
+
+                /* Optional timing tweak: */
+                // i2c-gpio,delay-us = <5>;
+
+                #address-cells = <1>;
+                #size-cells = <0>;
+                status = "okay";
+            };
```

Comment:
Technically, an unused bus without a device is considered invalid/dead code and shouldn't be present. It can be marked as `status = "disabled";` if it is a framework that another devicetree can then add a device sub-node and enable the bus.

That also depends on the device having a driver and not relying on `i2cdev`, so, if we are relying on a userspace application to talk to an i2cdev device then this is valid. But if we are connecting this to a kernel driver, and this PR is meant to fully introduce support for the device, then the device should be present as a subnode of the I2C bus here.